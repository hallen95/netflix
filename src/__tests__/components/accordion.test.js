import { render, fireEvent } from '@testing-library/react'
import faqsData from '../../fixtures/fqas.json'
import { Accordion } from '../../components'
import { Item } from '../../components/jumbotron/styles/jumbotron'

describe('<Accordion />', () => {
    it('it renders the <Accordion /> with populated data', () => {
        const { container, getByText } = render(
            <Container {...restProps}>
            <Inner>{children}</Inner>
        </Container>
        )
    })
})