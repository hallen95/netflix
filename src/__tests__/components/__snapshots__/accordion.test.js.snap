// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Accordion /> opens and closes the <Accordion /> component 1`] = `
<div
  class="accordion__Container-sc-13al13j-0 fXOcjn"
>
  <div
    class="accordion__Inner-sc-13al13j-2 dTRDWD"
  >
    <h1
      class="accordion__Title-sc-13al13j-3 kZpYCk"
    >
      Frequently Asked Questions
    </h1>
    <div
      class="accordion__Frame-sc-13al13j-1 dUVyFO"
    >
      <div
        class="accordion__Item-sc-13al13j-4 fHBLEs"
      >
        <div
          class="accordion__Header-sc-13al13j-5 iAQmzh"
        >
          What is Netflix?
          <img
            alt="Open"
            src="images/icons/add.png"
          />
        </div>
      </div>
      <div
        class="accordion__Item-sc-13al13j-4 fHBLEs"
      >
        <div
          class="accordion__Header-sc-13al13j-5 iAQmzh"
        >
          How much does Netflix cost?
          <img
            alt="Open"
            src="images/icons/add.png"
          />
        </div>
      </div>
      <div
        class="accordion__Item-sc-13al13j-4 fHBLEs"
      >
        <div
          class="accordion__Header-sc-13al13j-5 iAQmzh"
        >
          Where can I watch?
          <img
            alt="Open"
            src="images/icons/add.png"
          />
        </div>
      </div>
      <div
        class="accordion__Item-sc-13al13j-4 fHBLEs"
      >
        <div
          class="accordion__Header-sc-13al13j-5 iAQmzh"
        >
          How do I cancel?
          <img
            alt="Open"
            src="images/icons/add.png"
          />
        </div>
      </div>
      <div
        class="accordion__Item-sc-13al13j-4 fHBLEs"
      >
        <div
          class="accordion__Header-sc-13al13j-5 iAQmzh"
        >
          What can I watch on Netflix?
          <img
            alt="Open"
            src="images/icons/add.png"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Accordion /> renders the <Accordion /> with populated data 1`] = `
<div
  class="accordion__Container-sc-13al13j-0 fXOcjn"
>
  <div
    class="accordion__Inner-sc-13al13j-1 dKBwiU"
  >
    <h1
      class="accordion__Title-sc-13al13j-2 kGzmB"
    >
      Frequently Asked Questions
    </h1>
    <div
      class="accordion__Item-sc-13al13j-3 gvaOKT"
    >
      <div
        class="accordion__Header-sc-13al13j-4 eBcSJE"
      >
        What is Netflix?
        <img
          alt="Open"
          src="images/icons/add.png"
        />
      </div>
    </div>
    <div
      class="accordion__Item-sc-13al13j-3 gvaOKT"
    >
      <div
        class="accordion__Header-sc-13al13j-4 eBcSJE"
      >
        How much does Netflix cost?
        <img
          alt="Open"
          src="images/icons/add.png"
        />
      </div>
    </div>
    <div
      class="accordion__Item-sc-13al13j-3 gvaOKT"
    >
      <div
        class="accordion__Header-sc-13al13j-4 eBcSJE"
      >
        Where can I watch?
        <img
          alt="Open"
          src="images/icons/add.png"
        />
      </div>
    </div>
    <div
      class="accordion__Item-sc-13al13j-3 gvaOKT"
    >
      <div
        class="accordion__Header-sc-13al13j-4 eBcSJE"
      >
        How do I cancel?
        <img
          alt="Open"
          src="images/icons/add.png"
        />
      </div>
    </div>
    <div
      class="accordion__Item-sc-13al13j-3 gvaOKT"
    >
      <div
        class="accordion__Header-sc-13al13j-4 eBcSJE"
      >
        What can I watch on Netflix?
        <img
          alt="Open"
          src="images/icons/add.png"
        />
      </div>
    </div>
  </div>
</div>
`;
